integration_tests/workspace/.bazelversion

```diff
@@ -0,0 +1 @@
+7.0.0
```

Comment:
I'm not entirely sure how Bazelisk would handle that, although I'm happy to try. Tests may be run within the `integration_tests/workspace` directory as cwd so it would be reaching backwards up the directory hierarchy. `7.0.0` is also technically the default now, so we could also just delete it.